packages/help-center/src/types.ts

```diff
@@ -54,6 +54,7 @@ export interface FeatureFlags {
 }
 
 export interface SearchResult {
+	railcar: any;
```

Comment:
Can we at type it as: { fetch_algo: unknown, railcar: unknown, fetch_query: unknown, etc }? That way:


1. We assure there is a railcar, fetch_algo, etc. properties
2. We DENY any deeper access to railcar (eg, railcar.id or railcar.railcar.somethign)